packages/plugin-list/test/Checklist.test.ts

```diff
@@ -1269,16 +1279,21 @@ describePlugin(List, testEditor => {
                             contentBefore:
                                 '<ul class="checklist"><li>ab]cd</li><li style="list-style: none;"><ul class="checklist"><li class="unchecked">ef[gh</li></ul></li></ul>',
                             stepFunction: deleteForward,
-                            // falsy unchecked => because the title (all items of indented ul) is checked
+                            // The indented list's parent gets rendered as
+                            // checked because its only child is checked.
                             contentAfter:
                                 '<ul class="checklist"><li class="unchecked">ab[]gh</li></ul>',
                         });
                         await testEditor(BasicEditor, {
                             contentBefore:
                                 '<ul class="checklist"><li>ab]cd</li><li style="list-style: none;"><ul class="checklist"><li class="checked">ef[gh</li></ul></li></ul>',
                             stepFunction: deleteForward,
+                            // The indented list's parent gets rendered as
+                            // checked because its only child is checked. When
+                            // we remove that child, the checklist gets
+                            // unchecked because it becomes independant again.
```

Comment:
It's great that you talk about "becoming independant again" as we will be able to grep that to adapt the tests in case fp disagrees.